Makefile

```diff
@@ -13,9 +13,10 @@ clean:
 
 test:
 	cd Demo ; set -o pipefail && xcodebuild -workspace '$(WORKSPACE)' -scheme '$(SCHEME)' -configuration '$(CONFIGURATION)' test -sdk iphonesimulator -destination 'name=iPhone 5' | xcpretty -c --test
+	carthage build --no-skip-current
 
 setup:
-	cd Demo ; bundle install ; bundle exec pod install
+	cd Demo ; bundle install ; bundle exec pod install ; brew install carthage
```

Comment:
Hmm. Setup is really for internal development, which someone is totally able to do with or without Carthage. Dunno that it needs to be here though.